### PR TITLE
Some updates in descriptions

### PR DESCRIPTION
**Pull request by CruelKorleone**
I delete some unnecessary comments and added more descriptions

### CODE REVIEW

File: src/presentacion/FrmPrincipal.java [hunk 1]
```diff
@@ -1,8 +1,4 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
```
Comment (ozzbetto) [inline src/presentacion/FrmPrincipal.java:5]: Ok this is very important. Thanks

**Review by ozzbetto — COMMENTED** [review]
I think this is irrelevant.

**Review by ozzbetto — CHANGES_REQUESTED** [review]
I think this is a very irrelevant file. We can use the readme.md to add notes @CruelKorleone

**Comment by CruelKorleone** [comment]
OK Notes.md was deleted.

**Review by ozzbetto — APPROVED** [review]
Thanks, approved.

**Comment by ozzbetto** [comment]
Ok, Everything is fine merging.

**Pull request merged**